Modules/System/Cryptography Management/src/DESCryptoServiceProvider.Codeunit.al

```diff
@@ -42,14 +42,14 @@ codeunit 1379 DESCryptoServiceProvider
         DecryptedText := DESCryptoServiceProviderImpl.DecryptText(EncryptedText, Password, Salt);
     end;
 
-
     /// <summary>
     /// Encrypts data in stream with DotNet Cryptography.DESCryptoServiceProvider
     /// </summary>
     /// <param name="Password">Represents the password to be used to initialize a new instance of Rfc2898DeriveBytes</param>
     /// <param name="InputInstream">Represents the input instream data to encrypt</param>
     /// <param name="OutputOutstream">Represents the output instream encrypted data</param>
     [NonDebuggable]
+    [Obsolete('(Removed,Add the salt parameter to continue using this function)')]
     procedure EncryptStream(Password: Text; InputInstream: InStream; var OutputOutstream: Outstream)
     begin
         DESCryptoServiceProviderImpl.EncryptStream(Password, InputInstream, OutputOutstream);
```

Comment:
With the signature change in the implementation codeunit, this call is now missing the `Salt `parameter. You should use the password as the salt to ensure the same behavior as before.

```suggestion
        DESCryptoServiceProviderImpl.EncryptStream(Password, Password, InputInstream, OutputOutstream);
```